docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -33,98 +33,80 @@ refs:
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
+    - pattern: /docs/grafana-cloud/provision
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
 ---
 
 # Configure the Grafana Pyroscope data source
 
-To configure basic settings for the data source, complete the following steps:
-
-1. Click **Connections** in the left-side menu.
-1. Under Your connections, click **Data sources**.
-1. Enter `Grafana Pyroscope` in the search bar.
-1. Select **Add new data source**.
-1. Click **Grafana Pyroscope** to display the **Settings** tab of the data source.
-1. Set the data source's basic configuration options.
-1. Select **Save & test**.
-
-## Configuration options
+The Pyroscope data source sets how Grafana connects to your Pyroscope database.
 
-You can configure several options for the Pyroscope data source, including the name, HTTP, authentication, querying, and private data source connect.
+You can configure the data source using either the data source interface within Grafana or using a configuration file.
+This page explains how to set up and enable the data source capabilities using Grafana.
 
 If you make any changes, select **Save & test** to preserve those changes.
 
-![Configuration options for the Pyroscope data source](/media/docs/grafana/data-sources/screenshot-pyroscope-data-source-config.png)
-
-### Name and default
+If you are using your own installation of Grafana, you can provision the Pyroscope data source using a YAML configuration file.
+For more information about provisioning and available configuration options, refer to [Provisioning Grafana](ref:provisioning-data-sources).
 
-**Name**
-: Enter a name to specify the data source in panels, queries, and Explore.
+## Before you begin
 
-**Default**
-: The default data source is pre-selected for new panels.
+To configure a Pyroscope data source, you need administrator rights to your Grafana instance and a Pyroscope instance configured to send data to Grafana.
 
-### HTTP
+If you are provisioning a Pyroscope data source, then you also need administrative rights on the server hosting your Grafana instance.
 
-The HTTP section is shown in number 1 in the screenshot.
+## Set up the data source
 
-**URL**
-: The URL of the Grafana Pyroscope instance, for example, `https://localhost:4100`.
+To configure basic settings for the data source, complete the following steps:
 
-**Allowed cookies**
-: The Grafana Proxy deletes forwarded cookies. Use this field to specify cookies by name that should be forwarded to the data source.
+1. Click **Connections** in the left-side menu.
+1. Under Your connections, click **Data sources**.
+1. Enter `Grafana Pyroscope` in the search bar.
+1. Select **Add new data source**.
+1. Click **Grafana Pyroscope** to display the **Settings** tab of the data source.
+1. On the **Settings** tab, complete the **Name**, **Connection**, and **Authentication** sections.
```

Comment:
I'm assuming this procedure was lifted from other data sources, but this UI has changed since we had a data sources writer. I'm suggesting what I think is probably the happy path that the data sources folks would want, but you may want to double check this with them. 

Note that as soon as you click **Connections**, it'll take you straight to the **Add new connection** page. And when you click **Add new data source**, you're taken straight to the **Settings** tab of the new data source.

```suggestion
1. Enter `Grafana Pyroscope` in the search bar.
1. Select **Grafana Pyroscope**.
1. Click **Add new data source** in the top-right corner of the page.
1. On the **Settings** tab, complete the **Name**, **Connection**, and **Authentication** sections.
```

Reply:
his was part of the procedure that had existed, looked okay, but didn't verify. Thank you for checking!